Packs/Base/Scripts/DBotFindSimilarIncidents/DBotFindSimilarIncidents_test.py

```diff
@@ -1,8 +1,8 @@
 import demistomock as demisto
-import json
 import numpy as np
 import pandas as pd
 import pytest
+from copy import deepcopy
```

Comment:
the usage of deepcopy in the tests below makes them pass (the globals used in the tests are manipulated in the code, so sending a deep copy of them instead resolves that)